src/Visualization/Visualization.i

```diff
@@ -97,5 +98,14 @@ class Display3d {
 	Handle_AIS_InteractiveContext GetContext();
 	%feature("autodoc", "1");
 	void Test();
+
+	//%feature("autodoc", "1");
+    void ChangeRenderingParams(int RenderingMethod,
+                               int RaytracingDepth,
+                               bool IsShadowEnabled,

```

Comment:
hi @tpaviot  / @rainman110 , can't figure out how swig fails on this simple new method ( admittedly, I'm not a C++ programmer... ). what am I missing here?

```bash
Scanning dependencies of target _Visualization
[100%] Building CXX object CMakeFiles/_Visualization.dir/Darwin/x86_64-MinSizeRel-64/VisualizationPYTHON_wrap.cxx.o
/Users/jelleferinga/GIT/pythonocc-core/build/Darwin/x86_64-MinSizeRel-64/VisualizationPYTHON_wrap.cxx:4834:3: error: 
      unknown type name 'Display3d'
  Display3d *result = 0 ;
  ^
/Users/jelleferinga/GIT/pythonocc-core/build/Darwin/x86_64-MinSizeRel-64/VisualizationPYTHON_wrap.cxx:4841:17: error: 
      use of undeclared identifier 'Display3d'
      result = (Display3d *)new Display3d();
                ^
/Users/jelleferinga/GIT/pythonocc-core/build/Darwin/x86_64-MinSizeRel-64/VisualizationPYTHON_wrap.cxx:4841:28: error: 
      expected expression
      result = (Display3d *)new Display3d();
                           ^
/Users/jelleferinga/GIT/pythonocc-core/build/Darwin/x86_64-MinSizeRel-64/VisualizationPYTHON_wrap.cxx:4867:3: error: 
      unknown type name 'Display3d'
  Display3d *arg1 = (Display3d *) 0 ;
  ^
/Users/jelleferinga/GIT/pythonocc-core/build/Darwin/x86_64-MinSizeRel-64/VisualizationPYTHON_wrap.cxx:4867:22: error: 
      use of undeclared identifier 'Display3d'
  Display3d *arg1 = (Display3d *) 0 ;
                     ^
/Users/jelleferinga/GIT/pythonocc-core/build/Darwin/x86_64-MinSizeRel-64/VisualizationPYTHON_wrap.cxx:4867:33: error: 
      expected expression
  Display3d *arg1 = (Display3d *) 0 ;
                                ^
/Users/jelleferinga/GIT/pythonocc-core/build/Darwin/x86_64-MinSizeRel-64/VisualizationPYTHON_wrap.cxx:4878:28: error: 
      unknown type name 'Display3d'
  arg1 = reinterpret_cast< Display3d * >(argp1);
                           ^
/Users/jelleferinga/GIT/pythonocc-core/build/Darwin/x86_64-MinSizeRel-64/VisualizationPYTHON_wrap.cxx:4909:3: error: 
      unknown type name 'Display3d'
  Display3d *arg1 = (Display3d *) 0 ;
  ^
/Users/jelleferinga/GIT/pythonocc-core/build/Darwin/x86_64-MinSizeRel-64/VisualizationPYTHON_wrap.cxx:4909:22: error: 
      use of undeclared identifier 'Display3d'
  Display3d *arg1 = (Display3d *) 0 ;
                     ^
/Users/jelleferinga/GIT/pythonocc-core/build/Darwin/x86_64-MinSizeRel-64/VisualizationPYTHON_wrap.cxx:4909:33: error: 
      expected expression
  Display3d *arg1 = (Display3d *) 0 ;
                                ^
/Users/jelleferinga/GIT/pythonocc-core/build/Darwin/x86_64-MinSizeRel-64/VisualizationPYTHON_wrap.cxx:4930:28: error: 
      unknown type name 'Display3d'
  arg1 = reinterpret_cast< Display3d * >(argp1);
                           ^
/Users/jelleferinga/GIT/pythonocc-core/build/Darwin/x86_64-MinSizeRel-64/VisualizationPYTHON_wrap.cxx:4981:3: error: 
      unknown type name 'Display3d'
  Display3d *arg1 = (Display3d *) 0 ;
```